Packs/Code42/Integrations/Code42/Code42.yml

```diff
@@ -787,7 +787,7 @@ script:
     - contextPath: Code42.DepartingEmployee.DepartureDate
       description: The departure date for the Departing Employee.
       type: Unknown
-  dockerimage: demisto/py42:1.0.0.11140
+  dockerimage: demisto/py42:1.0.0.12043
```

Comment:
as written in the validation step, there is a new docker image:
Packs/Code42/Integrations/Code42/Code42.yml: [DO106] - The docker image tag is not the latest numeric tag, please update it.
The docker image tag in the yml file is: 1.0.0.12043
The latest docker image tag in docker hub is: 1.0.0.12174
please update to 1.0.0.12174 and update release notes to the newer verstion

Reply:
Yep, note that this was the latest when I opened the PR and another has come out since